ple/games/donkeykong/Coin.py

```diff
@@ -0,0 +1,46 @@
+__author__ = 'Batchu Vishal'
+import pygame
+from OnBoard import OnBoard
+
+'''
+This class defines all our coins.
+Each coin will increase our score by an amount of 'value'
+We animate each coin with 5 images
+A coin inherits from the OnBoard class since we will use it as an inanimate object on our board.
+'''
+
+
+class Coin(OnBoard):
+    def __init__(self, raw_image, position, rng):
+        super(Coin, self).__init__(raw_image, position, rng)
+        self.rng = rng
+        self.__coinAnimState = 0  # Initialize animation state to 0
+        self.IMAGES = {
+            "coin1": pygame.image.load('Assets/coin1.png'),
```

Comment:
performance: Might be useful to use image.convert() or image.convert_alpha() depending on if the image uses alpha